manifests/t/the-sz/Yale/1.21/the-sz.Yale.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+
+PackageIdentifier: the-sz.Yale
+PackageVersion: 1.21

```

Comment:
The package version does not appear in the ARP entry for this version. This will cause an upgrade loop
![image](https://user-images.githubusercontent.com/12611259/125690363-3b003c33-74a0-4d2c-add9-442dfd2532d9.png)

Reply:
fixed with new commit